modules/exploits/windows/fileformat/windows_script_host_vbscript.rb

```diff
@@ -200,7 +127,7 @@ def generate_vbscript(command_string, prepend_benign_code: false, prepend_new_li
     shell_obj = 'WScript.Shell'.chars.map { |c| (rand(2) == 0 ? c.downcase : c.upcase) }.join
     vbs_payload = "CreateObject(\"#{shell_obj}\").Run(\"#{cmd}\")"
     if obfuscate
-      vbs << obfuscate_vbscript(vbs_payload)
+      vbs << Rex::Exploitation::VBSObfuscate.new(vbs_payload).obfuscate!.to_s
```

Comment:
Was this meant to use the mixin? 🤔 

```suggestion
      vbs << vbs_obfuscate(vbs_payload)
```